charts/redpanda/templates/configmap.yaml

```diff
@@ -50,6 +50,15 @@ data:
   {{- if and (not (hasKey .Values.config.cluster "storage_min_free_bytes")) ((include "redpanda-atleast-22-2-0" . | fromJson).bool) }}
     storage_min_free_bytes: {{ include "storage-min-free-bytes" . }}
   {{- end }}
+{{- end }}
+{{- if and (include "is-licensed" .) .Values.storage.tieredConfig.cloud_storage_enabled }}
```

Comment:
"is-licensed" will return a string: "true" or "false" which will always test as true. This is why we use the fromJson/bool strategy.

Reply:
You are right. I follow your suggestion